pkg/compiler/debug.go

```diff
@@ -161,7 +164,7 @@ func (c *codegen) methodInfoFromScope(name string, scope *funcScope) *MethodDebu
 	return &MethodDebugInfo{
 		ID: name,
 		Name: DebugMethodName{
-			Name:      name,
+			Name:      strings.ToLower(string(name[0])) + name[1:],
```

Comment:
Probably not a popular case, but go allows to have multi-byte letters in function names. Not even sure we should support this, but this is not a hot path anyway.
https://goplay.tools/snippet/zHKtvoheyG1

Reply:
Nice example!